### PR TITLE
Escape quotes in namedplaces queries #3024

### PR DESCRIPTION
**Pull request by rafatower**
@Kartones please CR

### CODE REVIEW

**Comment by Kartones** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3248/))

**Comment by rafatower** [comment]
@Kartones can you please review again?

File: services/importer/lib/importer/namedplaces_guesser.rb [hunk 1]
```diff
@@ -102,6 +102,10 @@ def text_columns
           @text_columns ||= @guesser.columns.all.select{|c| @guesser.is_text_type?(c)}
         end
 
+        def sql_escape_quotes(str)
+          str.gsub("'", "''")
```
Comment (Kartones) [inline services/importer/lib/importer/namedplaces_guesser.rb]: Leave a TODO for the future when we migrate to ActiveRecord or something similar, no?

**Comment by Kartones** [comment]
:+1:

File: services/importer/lib/importer/namedplaces_guesser.rb (hunk 1 above)
Comment (rafatower) [inline services/importer/lib/importer/namedplaces_guesser.rb]: gonna use `ActiveRecord::Base.sanitize` and move it to the sql api class, where it belongs

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3252/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3254/))

**Pull request merged**